tests/test_index.py

```diff
@@ -855,6 +855,6 @@ def test_custom_storage_reopening(self) -> None:
         del r1
         self.assertTrue(storage.hasData)
 
-        r2 = index.Index(storage, properly=settings, overwrite=False)
+        r2 = index.Index(storage, properties=settings, overwrite=False)
```

Comment:
I suppose that the default properties were sufficient for this test to succeed. Maybe we should remove the properties for this test? It would remove L849-L851 and adjust this line and L853.